_format/1.1/index.md

```diff
@@ -631,6 +631,13 @@ The following characters **MUST NOT** be used in member names:
 - U+007F DELETE
 - U+0000 to U+001F (C0 Controls)
 
+#### <a href="#document-member-names-jsonld" id="document-member-names-jsonld" class="headerlink"></a> JSON-LD Exception
+
+When a JSON API document is [served as JSON-LD](http://www.w3.org/TR/json-ld/#interpreting-json-as-json-ld),
+the document **MAY** include member names that begin with an at sign
+(U+0040 COMMERCIAL AT, "@") to convey JSON-LD information. JSON API processers
```

Comment:
So, allowing "@" as a first character everywhere is "neater", in that it doesn't create a special case just for JSON LD. My thinking for writing it this way, though, was: 

1. It preserves the ability for us to use "@" in, e.g., a future query parameter's syntax.

2. Using an "@"-prefixed key for non-JSON LD data is a bad idea, because it means that if JSON LD adds a key of the same name in the future, your use may cause problems. This narrower allowance takes away that footgun.

2. It prevents "@" signs from creeping in to attribute, relationship, and other names, which we may not want for aesthetic reasons (seeing how much we've done to keep prefixes out), and which could be confusing to readers trying to distinguish JSON API data from JSON LD data.